waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -31,11 +45,12 @@ proc init*(T: type WakuMessageStore, db: SqliteDatabase): MessageStoreResult[T]
   let prepare = db.prepareStmt("""
     CREATE TABLE IF NOT EXISTS """ & TABLE_TITLE & """ (
         id BLOB PRIMARY KEY,
-        timestamp INTEGER NOT NULL,
+        receiverTimestamp BLOB NOT NULL,
```

Comment:
Does this require db migrations? What does a graceful upgrade path look like here?

Reply:
A short-term solution is to delete the existing db given that no important data would be lost. In the meanwhile, I am working on a decent migration approach using `user-version` pragma.